mdanalysissandbox/poem.py

```diff
@@ -0,0 +1,8 @@
+""" here is a docstring """
+
+def write_a_poem():
```

Comment:
Please add docstring.

Follow [MDA style guide](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#writing-docstrings)

Reply:
addressed in commit 679c2e3557970220e71af008f396f3bd4c8eef91